docs/Air.md

```diff
@@ -434,3 +435,18 @@ Gets pnr information and tickets list from [`importPNR`](#importPNR) and then do
 | ignoreTickets | `Boolean` | Defines if tickets should be ignored. The default value is `false` |
 
 **See: <a href="../examples/Air/cancelBooking.js">cancelBooking example</a>**
+
+## .addSegments(params)
```

Comment:
Please, add a statement, that version should be passed if user wants to make sure, that he had the last version overview before change. Could be like:
"It is recommended to pass `version` param in order to ensure that last version of the PNR was reviewed before modification".